digiwf-integrations/digiwf-dms-integration/digiwf-dms-integration-core/src/main/java/de/muenchen/oss/digiwf/dms/integration/domain/File.java

```diff
@@ -5,8 +5,7 @@
 
 @Data
 @AllArgsConstructor
-public class Content {
-
+public class File {
```

Comment:
In der Schnittstelle ist ein Schriftstück als "Content" und eine Sachakte als "File" bezeichnet, daher hatte ich das Schriftstück auch hier Content genannt.
Sollten wir hier das Schrifststück als File bezeichnen, müssen wir uns noch überlegen, wie wir die Sachakte dann benennen.

Reply:
du hast vollkommen recht, ich passe das schnell an.